pkg/config/multizone/kds.go

```diff
@@ -98,3 +103,22 @@ func (k KdsClientConfig) Sanitize() {
 func (k KdsClientConfig) Validate() error {
 	return nil
 }
+
+type ZoneHealthCheckConfig struct {
+	// PollInterval is the interval between the CP checking ZoneInsight for
```

Comment:
nit: 
```suggestion
	// PollInterval is the interval between the Global CP checking ZoneInsight for
```

Reply:
And at the same time, this is the interval between Zone CP pings. Should we add this to the comment?